### PR TITLE
Remove and uniformized function references

### LINKED ISSUE #568: Function should not be referred to with empty parenthesis
The documentation is currently not consistent when referring to function names, sometimes we find: `wb_robot_get_time()` and sometimes we find `wb_robot_get_time`. I believe we should systematically remove the `()` suffix as it is not a common usage in API docs and could be misleading as one could believe the function takes no argument.

### PR DESCRIPTION
**Pull request by fabienrohrer**
Fix #568

- Remove `()` notation.
- Add systematically the "function" suffix.

### CODE REVIEW

**Comment by fabienrohrer** [comment]
👍

**Comment by stefaniapedrazzi** [comment]
👀

File: reference/robot.md [hunk 1]
```diff
@@ -221,26 +222,26 @@ milliseconds, but the actuator commands could not be executed on time.
 and the sensor values were also measured at `controller_time` + `dt`. It means
 that the requested step duration could not be respected.
 
-The C API has two additional functions `wb_robot_init()` and
-`wb_robot_cleanup()`. There is not equivalent of the `wb_robot_init()` and
-`wb_robot_cleanup()` functions in the Java, Python, C++ and MATLAB APIs. In
+The C API has two additional functions: `wb_robot_init` and
+`wb_robot_cleanup`. There is not equivalent of the `wb_robot_init` and
```
Comment (stefaniapedrazzi) [inline reference/robot.md]: Old typo: `There is not equivalent` -> `There is no equivalent`?

File: reference/robot.md [hunk 2]
```diff
@@ -500,7 +501,7 @@ in the user guide).
 
 The integers can be compared to the following enumeration items:
 
-%figure "Helper enumeration to interpret the integer argument and return value of the *wb_robot_[gs]et_mode()* functions"
+%figure "Helper enumeration to interpret the integer argument and return value of the *wb_robot_[gs]et_mode* functions"
```
Comment (stefaniapedrazzi) [inline reference/robot.md]: Sometimes in figure caption the function is formatted using \*\* and sometimes with "\`<name>\`".
Is there a rule for this?

File: reference/supervisor.md [hunk 3]
```diff
@@ -1000,19 +1000,19 @@ void wb_supervisor_field_set_mf_string(WbFieldRef field, int index, const char *
...
+The set operations are received by Webots from possibly several supervisors running concurrently. In order to ensure reproducible simulation results, they are executed only once all set operations are received, just before advancing the simulation time. The order of execution of the set operations is defined by the order of the Supervisor nodes in the scene tree. As a consequence, if a supervisor sets the translation field of a node and immediately retrieves the absolute position of the same node using `wb_supervisor_node_get_position`, it will actually get the previous position of the node. This is because the execution of the set operation is postponed to the beginning of the next simulation step. In order to retrieve the new position of the node, a `wb_robot_step` call with a non-zero argument should be executed before calling `wb_supervisor_node_get_position`.
```
Comment (stefaniapedrazzi) [inline reference/supervisor.md]: In this sentences the modified functions have no `function` suffix (and no article).

**Comment by fabienrohrer** [comment]
@stefaniapedrazzi Thank you, I fixed your comments and more..

👍

**Pull request merged**